src/components/Layout/LayoutHeader.tsx

```diff
@@ -148,7 +176,49 @@ export const LayoutHeader = ({
             )}
           </div>
         </View>
+        {showTOC ? <TableOfContents headers={tocHeadings} /> : null}
```

Comment:
For the table of contents can we find a way to keep it where it is currently in the DOM? Putting it in the header makes it more difficult for keyboard navigators to get to on desktop, where as if it's still in the <main> element they can at least use the skip to main link